helix-view/src/document.rs

```diff
@@ -561,6 +562,11 @@ impl Document {
         Ok(())
     }
 
+    /// Replaces the text in the document
+    pub fn set_text(&mut self, text: Rope) {
+        self.text = text;
+    }
```

Comment:
Where/how is this used? If it's limited to tests only I'd `cfg(feature = "integration")` gate it because it skips all the hooks under `doc.apply()` so it won't update the LSP/syntax/etc

Reply:
Oh, this is a leftover from an early attempt to get the test document to its initial state, but it did not work for exactly the reason you said. I'll remove it.